### PR TITLE
Update OfficialUrls.xml

### PR DESCRIPTION
**Pull request by yusha1439**
added link for Arraed from http://arraedtv.technostreaming.net/smart.html

### CODE REVIEW

**Comment by yusha1439** [comment]
added link from http://arraedtv.technostreaming.net/smart.html

**Comment by Shani-08** [comment]
please check the xml again, it looks like we are openning the info tag again

File: OfficialUrls.xml [hunk 1]
```diff
@@ -1,4 +1,12 @@
 <streamingInfos>
+<streaminginfo>
+     <cname>Arraed</cname>
+	 <item>
+<title>m3u8_official_720p</title>
+<link>http://45.55.63.89:8081/arraedtv/abr/arraedtv/live_720p/chunks.m3u8</link>
+</item>
+</streaminginfo>
+<streaminginfo>
```
Comment (Shani-08) [inline OfficialUrls.xml]: not correct, is it? looks like a mistake

**Comment by Shani-08** [comment]
corrected the data. please always check if xml is valid.

**Comment by yusha1439** [comment]
what is wrong with it?

**Comment by Shani-08** [comment]
I have corrected it already. You he had streaminginfo opened twice at the end of your changes.

**Comment by yusha1439** [comment]
thanks, sorry for the mess

**Pull request merged**